docs/building/basic-build.md

```diff
@@ -2,10 +2,46 @@
 sidebar_position: 1
 ---
 
+# Basic instructions
+To compile release version of the smart contract you can run:
+
+```bash
+cargo build --target wasm32-unknown-unknown --release
+```
+
+> **Note:** The above `build` command is setting a `target` flag to create a WebAssembly `.wasm` file.
+
+Notice that your project directory now has a few additional items:
+
+```bash
+.
+├── Cargo.lock  ⟵ created during build to lock dependencies
+├── Cargo.toml
+├── src
+│  └── lib.rs
+└── target      ⟵ created during build, holds the compiled wasm
+```
 # Build and Flags
+We recommend you to optimize your build artifact with the use of the next flags.
+
+```bash
+env 'RUSTFLAGS=-C link-arg=-s' cargo build --target wasm32-unknown-unknown --release
+```
+
+The above command is essentially setting special flags and optimizing the resulting `.wasm` file. At the end of the day, it's simply a customized `cargo build --release` command.
+
+
+**Windows users**: please modify the above command as:
+```bash
+set RUSTFLAGS=-C link-arg=-s
+cargo build --target wasm32-unknown-unknown --release
+```
 
-<!-- Basic building instructions -->
+You can simpify this command by adding this flags to your `ProjectFolder/.cargo/config.toml`.
 
-<!-- Why we need flags and certain build configuration -->
+```toml
+[target.wasm32-unknown-unknown]
+rustflags = ["-C", "link-arg=-s"]
```

Comment:
```suggestion
rustflags = ["-C link-arg=-s"]
```

Note that this is something new in near-sdk-rs that Austin discovered is an improvement:
https://github.com/near/near-sdk-rs/pull/543/files#diff-8a51f1a3e37218978813e2fdb533fb9bbb6d687472e2a0a3f8673ba995de3054R2

Reply:
nah, this is the correct version. See https://github.com/near/near-sdk-rs/commit/05e4539a8f3db86dd43b768ee9660dd4c8e7ea5c.

The version you suggest doesn't work on some archs

Reply:
@mikedotexe yes, I took it from the `near-sdk-rs` repo. Please, resolve/merge if you agree.